diff --git a/node_modules/react-native-maps/android/build.gradle b/node_modules/react-native-maps/android/build.gradle
index 067a913..777ce13 100644
--- a/node_modules/react-native-maps/android/build.gradle
+++ b/node_modules/react-native-maps/android/build.gradle
@@ -25,6 +25,9 @@ def isNewArchitectureEnabled() {
 
 android {
   namespace "com.rnmaps.maps"
+  buildFeatures {
+    buildConfig true
+  }
   def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
   if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
     namespace "com.rnmaps.maps"
